vagrant/frontend/app/start.sh

```diff
@@ -9,4 +9,4 @@ export VUE_APP_REDIS_ENDPOINT=
 cd /home/vagrant/kuma-demo/frontend
 
 # Serve the application on localhost 
-forever start -c http-server dist -P http://127.0.0.1:23001
+forever start -c http-server dist -a 127.0.0.1 -P http://127.0.0.1:23001
```

Comment:
Sorry, I didn't understand your question.

Let's make it explicit that `http-server` is listening on port `8080`